### PR TITLE
Sos 569 run git

### PR DESCRIPTION
**Pull request by aylusltd**

### CODE REVIEW

File: src/verinfast/config.py [hunk 1]
```diff
@@ -229,6 +229,13 @@ def __init__(self, cfg_path: str = None) -> None:
             """
             self.modules = ConfigModules(code=cm, cloud=[])
             self.runGit = False
+        elif (
+            "repos" in self.config and
+            "modules" in self.config and
```
Comment (SeanTConrad) [inline src/verinfast/config.py:234]: what about local repos?

Comment (SeanTConrad) [reply src/verinfast/config.py:234]: ("repos" in self.config or "local_repos" in self.config)

**Review by SeanTConrad — CHANGES_REQUESTED** [review]

File: src/verinfast/config.py (hunk 1 above)
Comment (aylusltd) [reply src/verinfast/config.py:234]: shouldn't run git on local repos we don't clone those or change branches (since we can't change back)

Comment (SeanTConrad) [reply src/verinfast/config.py:234]: We lose a lot of widgets then if we run on local repo, so we shouldn't suggest it to targets.

**Review by SeanTConrad — APPROVED** [review]

**Pull request merged**